build/package/Installer.RPM.targets

```diff
@@ -0,0 +1,213 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project='Installer.RPM.props'/>
+
+  <UsingTask TaskName="ReplaceFileContents" AssemblyFile="$(CLIBuildDll)"/>
+  <UsingTask TaskName="BuildFPMToolPreReqs" AssemblyFile="$(CLIBuildDll)"/>
+
+  <PropertyGroup>
+    <BuildRpmPackage Condition=" '$(IsRPMBasedDistro)' == True and '$(Architecture)' == 'x64' ">True</BuildRpmPackage>
+  </PropertyGroup>
+
+  <Target Name="GenerateRpms"
+          DependsOnTargets="TestFPMTool;BuildRpms;TestSdkRpm"
+          Condition=" '$(BuildRpmPackage)'=='True' "
+          Outputs="@(GeneratedInstallers)"/>
+
+  <Target Name="BuildRpms"
+          DependsOnTargets="GenerateSharedHostRpm"
+          Condition=" '$(BuildRpmPackage)'=='True' and '$(FPMPresent)'=='True' "/>
```

Comment:
Ahh there is an comment. Jenkins don't have the tools
<!-- Workaround for Jenkins machines that don't have the necessary packages https://github.com/dotnet/core-setup/issues/2260 -->